database/postgres/index.js

```diff
@@ -0,0 +1,37 @@
+const { Pool } = require('pg');
```

Comment:
should we use 'Client' now, to measure initial performance and then switch to 'Pool' as an optimization?

Reply:
Probably, I'll try both when measuring performance.